scripts/rollup/build.js

```diff
@@ -155,8 +156,10 @@ function getBabelConfig(
     packageName === 'react' || externals.indexOf('react') !== -1;
   let options = {
     exclude: '/**/node_modules/**',
+    babelHelpers: 'bundled',
     babelrc: false,
     configFile: false,
+    sourceMaps: true,
```

Comment:
It looks like `@rollup/plugin-babel` generates sourcemaps by default since `2.1.0`:

- https://github.com/rollup/plugins/blob/master/packages/babel/CHANGELOG.md#210

So we probably don't need to add this (unless it's also going to other plugins that don't set it by default?)

```suggestion
```